javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -354,63 +212,370 @@
                     "version_added": "1.5"
                   },
                   "webview_android": {
-                    "version_added": "37"
+                    "version_added": "4.4"
                   }
                 },
                 "status": {
                   "experimental": false,
                   "standard_track": true,
                   "deprecated": false
                 }
-              }
-            },
-            "timeStyle": {
-              "__compat": {
-                "support": {
-                  "chrome": {
-                    "version_added": "76"
-                  },
-                  "chrome_android": {
-                    "version_added": "76"
-                  },
-                  "edge": {
-                    "version_added": "79"
-                  },
-                  "firefox": {
-                    "version_added": "79"
-                  },
-                  "firefox_android": {
-                    "version_added": "79"
-                  },
-                  "ie": {
-                    "version_added": false
-                  },
-                  "nodejs": {
-                    "version_added": "12.9.0"
-                  },
-                  "opera": {
-                    "version_added": "63"
-                  },
-                  "opera_android": {
-                    "version_added": "54"
-                  },
-                  "safari": {
-                    "version_added": "14.1"
-                  },
-                  "safari_ios": {
-                    "version_added": "14.5"
-                  },
-                  "samsunginternet_android": {
-                    "version_added": "12.0"
-                  },
-                  "webview_android": {
-                    "version_added": "76"
+              },
+              "options_dateStyle_parameter": {
```

Comment:
All of these are just renamed parameters and moved to subfeatures of the option.